app/Http/Controllers/ProjectController.php

```diff
@@ -101,8 +101,11 @@ public function store(Request $request)
 
     public function edit(Project $project)
     {
-        $languages = $project->languages;
+        if (! $this->isProjectOwner($project)) {
```

Comment:
Project model got a owner function. Now even if you want to check with the user admin, can we add it directly in the project model ? ( I think so) But in any case aren't those function related with the project should be in the model and not controller no ?